tests/unit/help_test.sh

```diff
@@ -3,6 +3,51 @@
 include './tests/unit/utils.sh'
 include './src/help.sh'
 
+function oneTimeSetUp()
+{
+  declare -g PATH_TO_GIT_REPOSITORY
+  declare -g PATH_TO_KW_REPOSITORY
+
+  PATH_TO_GIT_REPOSITORY="${SHUNIT_TMPDIR}/git_repo"
+  PATH_TO_KW_REPOSITORY=$(realpath "$(dirname "${BASH_SOURCE[0]}")"/../..)
+
+  setupGitRepository
+}
+
+function oneTimeTearDown()
+{
+  teardownGitRepository
+}
+
+# Execute a git command in our temporary git directory.
+function git_cmd()
+{
+  git -C "${PATH_TO_GIT_REPOSITORY}" "$@"
+}
+
+# We may want supress git's output so it doesn't show up when running tests.
+function git_cmd_silent()
+{
+  git_cmd "$@" > /dev/null 2>&1
+}
+
+# Copy KW to our temporary git repository and setup git credentials.
+function setupGitRepository()
+{
+  cp -r "${PATH_TO_KW_REPOSITORY}" "${PATH_TO_GIT_REPOSITORY}"
+  git_cmd_silent config user.name kw
+  git_cmd_silent config user.email kw@kw
```

Comment:
Maybe add the `--local` to avoid default behavior.

Reply:
btw, we have `mk_fake_git` in the `utils.sh`, can we leverage it here instead of creating all of these things?

Reply:
I did not know about `mk_fake_git`. I'll use it then.